app/config/FrontendAppConfig.scala

```diff
@@ -26,36 +26,37 @@ import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 @Singleton
 class FrontendAppConfig @Inject() (configuration: Configuration, servicesConfig: ServicesConfig) {
 
-  val host: String    = configuration.get[String]("host")
-  val appName: String = configuration.get[String]("appName")
+  lazy val host: String    = configuration.get[String]("host")
```

Comment:
I believe we can leave them as `val` They are unlikely to change, also making them lazy might lead to runtime issues if configs are missing.

Reply:
They have to be lazy for aid mocking. But the alternative would be to create a trait and then a module binding for it to a concreate class. So, I can update this to follow that pattern instead and then they can switch back to non-lazy.

I'll refactor this now 👍